File: example requests/requests.cs
// GET /fotoweb/

var client = new RestClient("https://foto.munchmuseet.no/fotoweb/");
client.Timeout = -1;
var request = new RestRequest(Method.GET);
request.AddHeader("Accept", "application/vnd.fotoware.api-descriptor+json");
IRestResponse response = client.Execute(request);
Console.WriteLine(response.Content);

// GET /fotoweb/archives/{archive}

var client = new RestClient("https://foto.munchmuseet.no/fotoweb/archives/5014-Malerier");
client.Timeout = -1;
var request = new RestRequest(Method.GET);
request.AddHeader("Accept", "application/vnd.fotoware.assetlist+json");
IRestResponse response = client.Execute(request);
Console.WriteLine(response.Content);

// GET /fotoweb/archives/{archive}/Arkiv/{filename}.info

var client = new RestClient("https://foto.munchmuseet.no/fotoweb/archives/5014-Malerier/Arkiv/M0112_20190403.tif.info");
client.Timeout = -1;
var request = new RestRequest(Method.GET);
request.AddHeader("Accept", "application/vnd.fotoware.asset+json");
IRestResponse response = client.Execute(request);
Console.WriteLine(response.Content);

GET /fotoweb/archives

var client = new RestClient("https://foto.munchmuseet.no/fotoweb/archives");
client.Timeout = -1;
var request = new RestRequest(Method.GET);
request.AddHeader("Accept", "application/vnd.fotoware.collectionlist+json");
IRestResponse response = client.Execute(request);
Console.WriteLine(response.Content);


GET /fotoweb/archives/{archive}

var client = new RestClient("https://foto.munchmuseet.no/fotoweb/archives/5014-Malerier");
client.Timeout = -1;
var request = new RestRequest(Method.GET);
request.AddHeader("Accept", "application/vnd.fotoware.collection+json");
IRestResponse response = client.Execute(request);
Console.WriteLine(response.Content);


GET /fotoweb/archives/{archive}

var client = new RestClient("https://foto.munchmuseet.no/fotoweb/archives/5014-Malerier");
client.Timeout = -1;
var request = new RestRequest(Method.GET);
request.AddHeader("Accept", "application/vnd.fotoware.collectioninfo+json");
IRestResponse response = client.Execute(request);
Console.WriteLine(response.Content);


GET /fotoweb/archives/{archive}/?{field name}={field content}

var client = new RestClient("https://foto.munchmuseet.no/fotoweb/archives/5014-Malerier/?105=vampire");
client.Timeout = -1;
var request = new RestRequest(Method.GET);
request.AddHeader("Accept", "application/vnd.fotoware.assetlist+json");
IRestResponse response = client.Execute(request);
Console.WriteLine(response.Content);

File: example requests/requests.js
// GET /fotoweb/

var settings = {
  "url": "https://foto.munchmuseet.no/fotoweb/",
  "method": "GET",
  "timeout": 0,
  "headers": {
    "Accept": "application/vnd.fotoware.api-descriptor+json"
  },
};

$.ajax(settings).done(function (response) {
  console.log(response);
});


// GET /fotoweb/archives/{archive}

var settings = {
  "url": "https://foto.munchmuseet.no/fotoweb/archives/5014-Malerier",
  "method": "GET",
  "timeout": 0,
  "headers": {
    "Accept": "application/vnd.fotoware.assetlist+json"
  },
};

$.ajax(settings).done(function (response) {
  console.log(response);
});


// GET /fotoweb/archives/{archive}/Arkiv/{filename}.info

var settings = {
  "url": "https://foto.munchmuseet.no/fotoweb/archives/5014-Malerier/Arkiv/M0112_20190403.tif.info",
  "method": "GET",
  "timeout": 0,
  "headers": {
    "Accept": "application/vnd.fotoware.asset+json"
  },
};

$.ajax(settings).done(function (response) {
  console.log(response);
});


// GET /fotoweb/archives

var settings = {
  "url": "https://foto.munchmuseet.no/fotoweb/archives",
  "method": "GET",
  "timeout": 0,
  "headers": {
    "Accept": "application/vnd.fotoware.collectionlist+json"
  },
};

$.ajax(settings).done(function (response) {
  console.log(response);
});


// GET /fotoweb/archives/{archive}

var settings = {
  "url": "https://foto.munchmuseet.no/fotoweb/archives/5014-Malerier",
  "method": "GET",
  "timeout": 0,
  "headers": {
    "Accept": "application/vnd.fotoware.collection+json"
  },
};

$.ajax(settings).done(function (response) {
  console.log(response);
});


// GET /fotoweb/archives/{archive}

var settings = {
  "url": "https://foto.munchmuseet.no/fotoweb/archives/5014-Malerier",
  "method": "GET",
  "timeout": 0,
  "headers": {
    "Accept": "application/vnd.fotoware.collectioninfo+json"
  },
};

$.ajax(settings).done(function (response) {
  console.log(response);
});


// GET /fotoweb/archives/{archive}/?{field name}={field content}

var settings = {
  "url": "https://foto.munchmuseet.no/fotoweb/archives/5014-Malerier/?105=vampire",
  "method": "GET",
  "timeout": 0,
  "headers": {
    "Accept": "application/vnd.fotoware.assetlist+json"
  },
};

$.ajax(settings).done(function (response) {
  console.log(response);
});
